server/eclipse-project/src/main/java/us/freeandfair/corla/model/CountyDashboard.java

```diff
@@ -545,6 +557,42 @@ public Integer estimatedBallotsToAudit() {
   }
   
   /**
+   * @return a String representation of this contest.
+   */
+  @Override
+  public String toString() {
+    return "CountyDashboard [county=" + id() + "]";
+  }
+
+  /**
+   * Compare this object with another for equivalence.
+   * 
+   * @param the_other The other object.
+   * @return true if the objects are equivalent, false otherwise.
+   */
+  @Override
+  public boolean equals(final Object the_other) {
```

Comment:
That would be very un-Java-conventional. And since Java doesn't support multiple inheritance, probably pretty messy, since the models don't currently all share the same base class.